bootstrap.sh

```
@@ -242,6 +241,195 @@ function getUbuntuVersion {
+    if [ -x "$(command -v timedatectl)" ]; then
+        runCmdAndLog timedatectl set-timezone ${TIMEZONE}
+    fi
+}
+
+function stepSetLocales() {
+    logStep "Setting locales to ${LOCALES[*]}..."
+    local LOCALE_GEN=locale-gen
+    if ! [ -x "$(command -v ${LOCALE_GEN})" ]; then
+        runCmdAndLog ${APT_INSTALL} locales
+    fi
+    runCmdAndLog ${LOCALE_GEN} ${LOCALES[@]}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)